code/modules/vore/eating/bellymodes_vr.dm

```diff
@@ -744,6 +769,214 @@
 						internal_contents -= P
 		return
 
+///////////////////////////// DM_TRANSFORM_REPLICA_EGG /////////////////////////////
+	if(digest_mode == DM_TRANSFORM_REPLICA_EGG && ishuman(owner))
+		for (var/mob/living/carbon/human/P in internal_contents)
+			if(P.stat)
+				continue
+			if(P.absorbed) //If they're absorbed, don't egg them
+				return
+			var/mob/living/carbon/human/O = owner
+
+			if (O.custom_species)
```

Comment:
They can perform TF's, as the "else" below on line 888 makes it so does the same thing, just without the custom_species name. That being said, looking back at this, it's _really_ cluttered and can be executed a lot better. Should have all these changed in a bit.